hathor/p2p/netfilter/matches.py

```diff
@@ -0,0 +1,133 @@
+from ipaddress import ip_network
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from hathor.p2p.netfilter.context import NetfilterContext
+
+
+class NetfilterMatch:
+    def match(self, context: 'NetfilterContext') -> bool:
+        raise NotImplementedError
+
+
+class NetfilterMatchAll(NetfilterMatch):
+    def match(self, context: 'NetfilterContext') -> bool:
+        return True
+
+
+class NetfilterMatchAnd(NetfilterMatch):
+    def __init__(self, a, b):
```

Comment:
For what I understood in the `match` method, `a` and `b` are `NetfilterMatch`, right?

Reply:
Yes, I think it's just missing type annotations.

Reply:
You're correct, @pedroferreira1. Just added the type annotations. Thanks!